connectors/presto/presto-snowflake/src/main/resources/modernizer/violations.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<modernizer>
+    <violation>
+        <name>java/lang/Class.newInstance:()Ljava/lang/Object;</name>
```

Comment:
Why is this forked?

Reply:
@tdcmeehan - The files presto-checks.xml and violations.xml are required for building the snowflake module as a standalone component. Is it acceptable to move these files to the root folder at this location: https://github.com/IBM/watsonx-data/tree/ebb1cc2bd820c5ed2479289f4c428c79a2a65e65/connectors/presto?

Reply:
Yes, let's at least move it to a common location so it's not redone for each connector.